### ISSUE TITLE
remarkPlugins with remarkGfm not working!?

### ISSUE THREAD

**Issue by melroy89** [andi23rosca/solid-markdown#38]
@andi23rosca  When trying to use markdown text with a markdown table:

```js
import remarkGfm from 'remark-gfm';
import { SolidMarkdown } from 'solid-markdown';


// Some code later:
return (
    <SolidMarkdown remarkPlugins={[remarkGfm]}>
     {md()}
    </SolidMarkdown>
  );
// I leave out the rest of the code, but I think it should be clear
````

We are getting: `props.node is undefined`...

Could you please try to create a new NPM release??

![start-screenshot](https://github.com/user-attachments/assets/3a07006d-72f4-41e4-9a12-7cff9d85ef5a)

**Comment by jwr1** [comment]
Also, just to give a little more context, after some testing, it seems the error only happens with markdown that contains tables inside.

We've tested with both the latest npm release and the latest commit on GitHub, and it seems like a fix was made at some point that was not yet released on GitHub, since using the GitHub version renders fine. My guess was that it was this commit: a051a8aa58f75b53004da0cdc4972303a3731cdb.

**Comment by melroy89** [comment]
> I thought I had set up automatic builds for this library, just released `v2.0.14` with the latest, let me know if that fixes things 👍🏻

It does not seem to crash/error anymore anymore, however in my case I have an issue that it doesn't show me any markdown/html anymore, but I blame me system for now 😖 .

So you might want to wait until @jwr1 also says its fine.

**Comment by jwr1** [comment]
Yup, that fixed the issue.
